### ISSUE TITLE
[BUG] Opacity and colors changes when blur is activated

### ISSUE THREAD

**Issue by sebascontre** [FiniteSingularity/obs-composite-blur#91]
**Describe the bug**
Not sure if its a bug, but I think in reported in case it is one. When working with semi-transparent sources, and apply the blur filter, I notice some colors changes a little, and this is augmented when multiple blurs are activated.

**To Reproduce**
1. Create a semi transparent source. (In the example I use a a color one)
2. Activate the blur filter
3. Notice the opacity and/or colors change a little.

**Expected behavior**
To not change the opacity and/or colors?

**Screenshots**
![image](https://github.com/FiniteSingularity/obs-composite-blur/assets/1657758/e05bce2f-9e21-4517-b930-7bcb043499ba)

**Environment**
 - OS/Version: Windows 11
 - OBS Version: 30.0.0
 - Composite Blur Plugin Version: 1.0.6
 - Blur Algorithm/Type: All

**GPU**
 - GPU make/model: Nvidia GTX 1650
 - GPU VRAM: 4Gb

**Comment by FiniteSingularity** [comment]
@sebascontre thank you for opening the issue.  I'm just finishing up v1.0 of the Advanced Mask plugin, and will take a look at this later this week.

**Comment by FiniteSingularity** [comment]
So I've started diving into this, and there is definitely a bug with how alpha is being blended after the filter is applied.  I'm also seeing the same bug with the Advanced Mask plugin, and figure it has something to do with how I am grabbing the incoming texture.

**Comment by FiniteSingularity** [comment]
The PR above should have fixed this.  Once it is done building, I'll post a link to the compiled binaries that you can grab and ensure it fixes your issue. It'll all be included in the next release which I hope to get out this weekend.  If you are still having issues with this, please feel free to re-open the issue.

**Comment by FiniteSingularity** [comment]
@sebascontre here is a link to the actions run.  If you scroll to the bottom of that page, you should see installers and zip files, which you can try installing to see if it fixes your issue. https://github.com/FiniteSingularity/obs-composite-blur/actions/runs/7152890943

**Comment by FiniteSingularity** [comment]
oops!  Nevermind.. this broke a bunch of stuff.  Still in progress.

**Comment by FiniteSingularity** [comment]
I think we've got this one fixed.  Will post a link to the binaries/installer once the build script is complete.  If you can give it a try @sebascontre, I'd appreciate feedback on if it solves your problem.

**Comment by sebascontre** [comment]
I tried yesterday the build from workflow "[Attempt to fix the color consistency issue (#93)](https://github.com/FiniteSingularity/obs-composite-blur/actions/runs/7197895886)" and works great with the colors, but has an issue where masks doen't work. Later I tried the build from workflow "[Removes Background Source For Compositing Option.](https://github.com/FiniteSingularity/obs-composite-blur/actions/runs/7198978992)" and works amazingly colors doens't change, and mask works again!!

But, and just for to you to know, now OBS throws an error when trying to change the scene collection, not the end of the world, I can work around that, but best to told you just in case.

Thanks again for all of your support and time!!

![image](https://github.com/FiniteSingularity/obs-composite-blur/assets/1657758/f175cedb-a64c-49c6-b601-f20e5efe05df)

**Comment by FiniteSingularity** [comment]
Just saw this message.  I saw that closing bug, and it should be fixed now (was a scene that wasn't being released by the plugin).

I also had to make a bunch of changes to the filtering pipeline to get the background compositing working again, and I think I've got it all in good shape.  Colors seem good, compositing seems good, and no more errors on quit.  Feel free to grab the latest and let me know if it smooths things out for you.

https://github.com/FiniteSingularity/obs-composite-blur/actions/runs/7214220861
